src/service/action-impl.js

```diff
@@ -152,11 +158,39 @@ export class ActionService {
           this.trigger(dev().assertElement(event.target), 'tap', event);
         }
       });
-    } else if (name == 'submit' || name == 'change') {
+    } else if (name == 'submit') {
       this.root_.addEventListener(name, event => {
         this.trigger(dev().assertElement(event.target), name, event);
       });
+    } else if (name == 'change') {
+      this.root_.addEventListener(name, event => {
+        const target = event.target;
+        const changeEvent = this.getChangeDetails_(event);
+        this.trigger(dev().assertElement(target), name, changeEvent);
+      });
+    }
+  }
+
+  /**
+   * Generate custom event from browser `change` event.
+   * @param {!Event} event A `change` event.
+   * @return {!Event} A custom event with a details property containing the
+   *    relevant information for the change that generated the initial event.
+   */
+  getChangeDetails_(event) {
+    const details = {};
+    const target = event.target;
+    if (event.target.tagName.toLowerCase() === 'input') {
```

Comment:
I'm not sure that can be done in a clean way. For the `input` tag we need to switch on the input type, but I don't think we'll need to do that for the others. Maybe it would be best to reevaluate when we implement support for `<select>` and `<textarea>`?